### DISCUSSION TITLE
How to do this with swiper?

### DISCUSSION THREAD

**Discussion by abdukulov-talgat** [nolimits4web/swiper#5644] [Q&A] [answered]
![slider](https://user-images.githubusercontent.com/102265819/164044096-fcb1cd72-f85b-4a5a-b588-30c40f7ccc5a.png)
tried do it with coverflow, but its look different.

**Comment by Arhell** [comment, marked as answer]
Here are a few examples, the rest can be completed
https://codepen.io/tbego/pen/LrozoY
https://codepen.io/etuctida/pen/BLQNoO

**Reply by abdukulov-talgat** [reply]
Thx for help
